src/_cffi_src/openssl/dh.py

```diff
@@ -117,6 +117,10 @@
 #endif
 
 #if CRYPTOGRAPHY_OPENSSL_LESS_THAN_110 || defined(LIBRESSL_VERSION_NUMBER)
+# define DH_CHECK_Q_NOT_PRIME            0x10
```

Comment:
These should each be ifndef for safety since we don't want to fail with duplicate definitions if/when LibreSSL adds these.